### PR TITLE
Document RGW quota cache options

### PR DESCRIPTION
**Pull request by dang**
Quota cache options were undocumented.  Issue #18747

Signed-off-by: Daniel Gryniewicz <dang@redhat.com>

### CODE REVIEW

**Comment by ktdreyer** [comment]
Mind updating the commit message to use the standard? When commit messages refer to bare numbers like that, GitHub associates it with PRs :(

`Refs: http://tracker.ceph.com/issues/18747`

**Comment by vumrao** [comment]
@dang default is very high for option `rgw_user_quota_sync_interval`.

~~~
OPTION(rgw_user_quota_sync_interval, OPT_INT, 3600 * 24) // time period for accumulating modified buckets before syncing entire user stats
~~~

and I have tested with the setting value of options `rgw_bucket_quota_ttl` and `rgw_user_quota_bucket_sync_interval` as 0 and it worked fine.

**Comment by vumrao** [comment]
I have tested all three options:
~~~
rgw_bucket_quota_ttl = 0
rgw_user_quota_bucket_sync_interval = 0
rgw_user_quota_sync_interval = 0
~~~
 as value zero and have same effect. Quota is working properly with two RGW instances in HA Proxy round-robin.  So good to have all three marked as zero.

I think we just need to change the default value of `rgw_user_quota_sync_interval` from `180` to `3600*24`.

File: doc/radosgw/config-ref.rst [hunk 1]
```diff
@@ -352,6 +352,35 @@ Ceph configuration file, the default value will be set automatically.
 :Default: ``false``
 
 
+``rgw bucket quota ttl``
+
+:Description: The amount of time in seconds cached quota information it
```
Comment (cbodley) [inline doc/radosgw/config-ref.rst]: > quota information it

quota information is

**Comment by mattbenjamin** [comment]
@dang  update for typo?

**Comment by cbodley** [comment]
ping @dang

**Comment by tchaikov** [comment]
@dang ping?

**Pull request merged**